### PR TITLE
Confirm rejudging on submission page

### PR DESCRIPTION
**Pull request by Riolku**
If a contest sub is rejudged and the verdict doesn't change, did the organizer still mess up?

### CODE REVIEW

File: templates/submission/status.html [hunk 1]
```diff
@@ -74,10 +74,19 @@
         <div><a href="{{ url('problem_submit', submission.problem.code, submission.id) }}">{{ _('Resubmit') }}</a></div>
     {% endif %}
     {% if perms.judge.rejudge_submission and not submission.is_locked %}
+        {% compress js %}
```
Comment (quantum5) [inline templates/submission/status.html:77]: Why are we shoving this into a separate js file?

Comment (Riolku) [reply templates/submission/status.html:77]: Where should it go? The JS blocks in this file are in unrelated if blocks.

Comment (Xyene) [reply templates/submission/status.html:77]: Eh, seems fine, it'll get compressed.

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

**Pull request merged**